clients/drcachesim/tests/offline-interval-count-output.templatex

```diff
@@ -1,61 +1,61 @@
            0 total physical address unavailable markers
-     .* total other markers
-     .* total encodings
-Thread .* counts:
-     .* \(fetched\) instructions
-     .* unique \(fetched\) instructions
-     .* non-fetched instructions
-     .* prefetches
-     .* data loads
-     .* data stores
-     .* icache flushes
-     .* dcache flushes
-     .* scheduling markers
-     .* transfer markers
-     .* function id markers
-     .* function return address markers
-     .* function argument markers
-     .* function return value markers
+     [ 0-9]* total other markers
+     [ 0-9]* total encodings
+Thread [0-9]* counts:
+     [ 0-9]* \(fetched\) instructions
+     [ 0-9]* unique \(fetched\) instructions
+     [ 0-9]* non-fetched instructions
+     [ 0-9]* prefetches
+     [ 0-9]* data loads
+     [ 0-9]* data stores
+     [ 0-9]* icache flushes
+     [ 0-9]* dcache flushes
+     [ 0-9]* scheduling markers
+     [ 0-9]* transfer markers
+     [ 0-9]* function id markers
+     [ 0-9]* function return address markers
+     [ 0-9]* function argument markers
+     [ 0-9]* function return value markers
            0 physical address \+ virtual address marker pairs
            0 physical address unavailable markers
-     .* other markers
-     .* encodings
+     [ 0-9]* other markers
+     [ 0-9]* encodings
 Counts per trace interval for whole trace:
-Interval #1 ending at timestamp.*
-     .* interval delta \(fetched\) instructions
-     .* interval delta non-fetched instructions
-     .* interval delta prefetches
-     .* interval delta data loads
-     .* interval delta data stores
-     .* interval delta icache flushes
-     .* interval delta dcache flushes
-     .* interval delta scheduling markers
-     .* interval delta transfer markers
-     .* interval delta function id markers
-     .* interval delta function return address markers
-     .* interval delta function argument markers
-     .* interval delta function return value markers
+Interval #1 ending at timestamp [0-9]*:
+     [ 0-9]* interval delta \(fetched\) instructions
+     [ 0-9]* interval delta non-fetched instructions
+     [ 0-9]* interval delta prefetches
+     [ 0-9]* interval delta data loads
+     [ 0-9]* interval delta data stores
+     [ 0-9]* interval delta icache flushes
+     [ 0-9]* interval delta dcache flushes
+     [ 0-9]* interval delta scheduling markers
+     [ 0-9]* interval delta transfer markers
+     [ 0-9]* interval delta function id markers
+     [ 0-9]* interval delta function return address markers
+     [ 0-9]* interval delta function argument markers
+     [ 0-9]* interval delta function return value markers
            0 interval delta physical address \+ virtual address marker pairs
            0 interval delta physical address unavailable markers
-     .* interval delta other markers
-     .* interval delta encodings.*
+     [ 0-9]* interval delta other markers
+     [ 0-9]* interval delta encodings
 .*
```

Comment:
I think you reviewed a previous version. This PR's latest version has the same change as a6772a2.

I found that in some runs on AArch64 there was only a single interval. Expecting a `.*` on a new line failed such runs. We need a `.*` at the end of the last line, and not on a new line.

> I believe the only change needed is the one in my commit 

There are some other changes too in this PR.

Reply:
> I think you reviewed a previous version. 

I reviewed the one sent for review -- ok you've updated it afterward.

> I found that in some runs on AArch64 there was only a single interval. Expecting a .* on a new line failed such runs. We need a .* at the end of the last line, and not on a new line.

That is what I was saying and is what is in my commit.

OK let's go with this one then since I had not had time to split mine out (was out most of the morning/afternoon) and you want to add tighter matches on the other lines.

Reply:
> I reviewed the one sent for review -- ok you've updated it afterward.

Yes. The AArch64 one showed up only after the CI run. Maybe should've waited for it to complete before requesting review.

> you want to add tighter matches on the other lines.

Right. That was the other bug that this PR fixes.